actiontext/lib/templates/installer.rb

```diff
@@ -8,6 +10,10 @@
 JS_DEPENDENCIES = JS_PACKAGE["peerDependencies"].dup.merge \
   JS_PACKAGE["name"] => "^#{JS_PACKAGE["version"]}"
 
+say "Installing JavaScript dependencies"
+run "bin/yarn add #{JS_DEPENDENCIES.map { |name, version| "#{name}@#{version}" }.join(" ")}",
+    abort_on_failure: true
```

Comment:
If `bin/yarn` does not exist, I would like to emit an error message that shows it. 
Can you add `capture: true` option?

Reply:
@y-yagi. I just added the `capture: true` option. However, I'm afraid that some of the work in this PR overlaps what is being developed here #35085.

Would you please read the questions I left in #35085? Do you think we should keep this installer just for the javascript dependencies?

I am not sure if the generator created in #35085 should deal with the javascript dependencies and run yarn install or if these two would be separate. Do you think we can finish the work on this PR and invoke this installer from the generator?